src/common/util.cpp

```diff
@@ -465,10 +465,18 @@ std::string get_nix_version_display_string()
       pathRet = "/";
     else
       pathRet = pszHome;
-#ifdef MAC_OSX
+#ifdef __APPLE__
```

Comment:
`__APPLE__` will match iOS too. Not aware if this file is used by the iOS port but worth mentioning any Mac checks should be explicit about OSX.

Reply:
I could specifically target OSX
```C++
#include "TargetConditionals.h"
#ifdef TARGET_OS_MAC
```
But is this really required? I'd like @moneromooo-monero 's point of view here.

Reply:
Looking at the cmake files there is nothing excluding this for iOS builds so it looks like it definitely needs the proper check.
You'll need to wrap the include of TargetConditionals in __APPLE__ and then check TARGET_OS_MAC.

Reply:
👍 Has been updated.

Reply:
I have no idea about Mac. Please use what seems best :)